pages.de/linux/amixer.md

```diff
@@ -0,0 +1,12 @@
+# amixer
+
+> Mixer für den ALSA Soundkarten Treiber.
+> Mehr Informationen: <https://manned.org/amixer>.
+
+- Erhöhe die Haupt Lautstärke um 10%:
```

Comment:
Not sure, but how about `Gesamtpegel`? At least this seems to be the correct translation. <https://dict.leo.org/german-english/master%20volume>